cli/command/context/testdata/list-json.golden

```diff
@@ -0,0 +1,5 @@
+{"Current":false,"Description":"description of context1","DockerEndpoint":"https://someswarmserver.example.com","Error":"","Name":"context1"}
```

Comment:
Given the TestListJSON test above, shouldn't context1-2-3 contain the additional `Type:` metadata?

Reply:
Not currently, no; the JSON output in list format doesn't output the "raw" metadata, and doesn't include custom fields currently.

_Should_ it include these? Maybe, maybe not (at least `inspect` shows them), but it's currently expected for them to not be here.